app/src/main/java/org/mozilla/fenix/tabstray/syncedtabs/SyncedTabs.kt

```diff
@@ -103,7 +138,11 @@ fun SyncedTabsList(syncedTabs: List<SyncedTabsListItem>, onTabClick: (SyncTab) -
  */
 @Composable
 fun SyncedTabsDeviceItem(deviceName: String) {
-    Column(Modifier.fillMaxWidth()) {
+    Column(
+        modifier = Modifier
+            .fillMaxWidth()
+            .background(FirefoxTheme.colors.layer1)
```

Comment:
Wondering if this is really needed since i didn't notice any difference in my testing.

Reply:
Without the background color, the text is in a transparent box, which causes the synced tabs' text to be visible underneath it as it scrolls